.github/pull-request-template.md

```diff
@@ -1,40 +1,41 @@
 ## **Description**
-_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
-_1. What is the reason for the change?_
-_2. What is the improvement/solution?_
+
+<!--
+Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
+1. What is the reason for the change?
+2. What is the improvement/solution?
+-->
+
+## **Related issues**
```

Comment:
Hey @georgewrmarshall great simplification, was just putting myself into normal creating MR flow, usually i would specify the issue/ ticket number i'm fixing first, then adding description ~ wdyt?

Reply:
Hi @DDDDDanica , I understand your point. The initial reason though why we decided to place the description in first position with @NicolasMassart was because it gets nicely previewed when the issue is posted on Slack.

Reply:
The suggestion to include the issue number at the beginning of a PR is valid, but considering @gauthierpetetin's point about PR previews when linked elsewhere, it's beneficial to have it second. This placement might also motivate engineers to provide a more comprehensive description, rather than merely linking the issue.